Gemfile

```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 gemspec
 
 gem 'jruby-openssl', platforms: :jruby
-gem 'rubocop', '~> 0.60.0', require: false
+gem 'rubocop', '~> 0.62.0', require: false
```

Comment:
Bumped up `rubocop` to mitigate error with `ruby 2.6.6p146` and `bundler 2.1.4`.

```ruby
unknown keywords: whitelist_classes, whitelist_symbols
```

Reply:
Rubocop bump looks good. I know some people have recently had issues with this, so hopefully this resolves that 🤞